DataConnectors/GCP/README.md

```diff
@@ -0,0 +1,56 @@
+# GCP Microsoft Sentinel Connector
+
+## Introduction
+
+GCP Sentinel connector is a platform for ingesting GCP service logs into Azure Sentinel. Currently supported logs include: Audit logs.
+
+This connector requires that each GCP service will publish its logs to a pub/sub. In addition you must configure workload identity pool and workload identity provider with permissions for the connector to receive an access token as well as a service account with permissions for the connector to read the logs from the pub/sub subscription.
+
+More information on the connector and configuration instructions can be found on the Azure Sentinel data connector page in the Azure portal.
```

Comment:
Maybe provide link?

Reply:
It's a deep link to workspace that already installed the connector so I don't think it will work for everyone.